extensions/ql-vscode/src/common/vscode/progress.ts

```diff
@@ -108,6 +108,10 @@ export function withInheritedProgress<R>(
   }
 }
 
+export function readableBytesMb(numBytes: number): string {
```

Comment:
(Optional) Would it make sense to move this function out of `progress.ts`? It feels like it belongs elsewhere (e.g. common/bytes.ts or something like that).